benchmark/buffers/buffer-read-float.js

```diff
@@ -3,7 +3,7 @@ const common = require('../common.js');
 
 const bench = common.createBenchmark(main, {
   type: ['Double', 'Float'],
-  endian: ['BE', 'LE'],
+  endian: ['LE'],
```

Comment:
The read float functions work identical, just counting them up or down. Thus the endian should not have any influence.